app/(protected)/(bookList)/dialog/content-form/handler.ts

```
@@ -0,0 +1,91 @@
+        shouldValidate: true,
+      });
+    }
+    field.onChange(e);
+  };
+
+export const onNumberValueChange =
+  (field: ControllerRenderProps<ContentUpdateForm, "nQuestion" | "nOptions">) =>
+  (e: React.ChangeEvent<HTMLInputElement>) => {
+    const val = parseInt(e.target.value);
+    const isNegative = val < 0;
+    field.onChange(isNaN(val) || isNegative ? 0 : val);
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)